configurations/local.template.ini

```diff
@@ -137,8 +137,8 @@ metadata_sphinx_num_of_int_fields = 10
 ;the range is used when checking service actions permissions and allowing to access certain 
 ;services without KS from the internal servers
 
-partner_otp_internal_ips = @IP_RANGE@
-otp_required_partners[] = -2
+;partner_otp_internal_ips = @IP_RANGE@
```

Comment:
@yossipapi in that case the OTP box will still be shown, and there are cases it prevents logging in. Better to have it disabled by default and use the SaaS local.ini (that already has these settings) to set this for SaaS only.